dcat/index.html

```diff
@@ -1872,6 +1876,94 @@ <h4>Property: was generated by</h4>
+            <a href="#Property:resource_has_policy">has policy</a>,
+            <a href="#Property:resource_identifier">identifier</a>,
+            <a href="#Property:resource_is_referenced_by">is referenced by</a>,
+            <a href="#Property:resource_keyword">keyword/tag</a>,
+            <a href="#Property:resource_landing_page">landing page</a>,
+            <a href="#Property:resource_license">license</a>,
+            <a href="#Property:resource_language">resource language</a>,
+            <a href="#Property:resource_relation">relation</a>,
+            <a href="#Property:resource_rights">rights</a>,
+            <a href="#Property:resource_qualified_relation">qualified relation</a>,
+            <a href="#Property:resource_publisher">publisher</a>,
+            <a href="#Property:resource_release_date">release date</a>,
+            <a href="#Property:resource_theme">theme/category</a>,
+            <a href="#Property:resource_title">title</a>,
+            <a href="#Property:resource_type">type/genre</a>,
+            <a href="#Property:resource_update_date">update/modification date</a>,
+            <a href="#Property:resource_qualified_attribution">qualified attribution</a>. 
+            <a href="#Property:dataset_frequency">frequency</a>,
+            <a href="#Property:dataset_spatial">spatial/geographic coverage</a>,
+            <a href="#Property:dataset_spatial_resolution">spatial resolution</a>,
+            <a href="#Property:dataset_temporal">temporal coverage</a>,
+            <a href="#Property:dataset_temporal_resolution">temporal resolution</a>,
+            <a href="#Property:dataset_was_generated_by">was generated by</a>.
+        </p>
+        <aside class="issue" title="Editorial questions">
+              <p>
+                  Does being ADMS a non-normative namespace prevent to list <code>adms:next</code>, <code>adms:prev</code> in this section?
+            </p>
+            <p>
+             Should restate the properties whose values are involved in dataset series inheritance e.g. <code>dct:issued</code>,<code>dct:spatial</code> explaining about contraints in their usage notes?
+             </p>
+           </aside>
+    
+        <table class="definition">
+            <thead><tr><th>RDF Class:</th><th><a href="http://www.w3.org/ns/dcat#DatasetSeries">dcat:DatasetSeries</a></th></tr></thead>
+            <tbody>
+            <tr><td class="prop">Definition:</td><td>A collection of datasets that are published separatelly, but are sharing common characteristic and could be made available as a single dataset.</td></tr>
+            <tr><td class="prop">Sub-class of:</td><td><a href="#Class:Dataset"><code>dcat:Dataset</code></a></td></tr>
+            <tr><td class="prop">Usage note:</td><td> Dataset series can be also soft-typed via property <code>dct:type</code> as in the approach used in [[GeoDCAT-AP]], and adopted in [[DCAT-AP-IT]] and [[GeoDCAT-AP-IT]]).</td></tr>
```

Comment:
We could also link to the DCAT-AP recommendation: https://joinup.ec.europa.eu/release/dcat-ap-how-model-dataset-series

Reply:
Thanks, Alejandra, 
apparently, apart from the link that you have attached, there is no occurrence of "series" in the lastest DCAT-AP, see document served at  https://joinup.ec.europa.eu/collection/semantic-interoperability-community-semic/solution/dcat-application-profile-data-portals-europe/release/201-0

I wouldn't refer to [[DCAT-AP]] unless we find a specific part in the "official document" dealing with it. What do you think?

Reply:
Makes sense, @riccardoAlbertoni - thank you